README.md

```diff
@@ -344,4 +389,4 @@ todo(aaron): find the appropriate license
 
 # Contact
 
-Please contact [Aaron R. Williams](awilliams@urban.org) with questions. 
+Please contact [Aaron R. Williams](awilliams@urban.org) with questions.
```

Comment:
I guess this is public, so we don't need to reference the slack channel here, but I also worry about all of these references to just Aaron.